test/test.js

```diff
@@ -255,4 +264,41 @@ describe('request-language', function() {
     };
     expect(method).to.throw('You haven\'t defined the markup `{language}` in your cookie.url settings.');
   });
+
+  it('should set the language requested in the query string', function() {
+    var middleware = requestLangauge({
```

Comment:
Oh, I see it now. I just copied and pasted existing examples, so I didn't even notice it was spelled wrong. It's spelled wrong in the import,  as well, which is why the code works anyway. =)